.github/workflows/test.yml

```diff
@@ -6,16 +6,24 @@ name: Run Service Tests
 on:
   push:
     branches: [ dev ]
+    paths-ignore:
+      - 'README.md'
+      - '.github/**'
+      - 'service/local-dev/**'
   pull_request:
     branches: [ '**' ]
```

Comment:
should this pattern exclude dev? Or can dev never be the right side of a pull request?